src/Calendar.js

```diff
@@ -112,6 +112,27 @@ class Calendar extends React.Component {
      */
     events: PropTypes.arrayOf(PropTypes.object),
 
+    /**
+     * An array of background event objects to display on the calendar. B events objects
```

Comment:
can we describe _what_ a background event is and how it's distinct from any other type of event?